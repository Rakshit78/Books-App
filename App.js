import { Platform, SafeAreaView, StyleSheet } from 'react-native';
import MyStack from './navigation';
import { NavigationContainer } from '@react-navigation/native';
import GlobalContext from './context';
import { useState } from 'react';
export default function App() {
  const [favData, setFavData] = useState([]);
  return (
    <SafeAreaView
      style={[styles.container, { marginTop: Platform.OS === 'ios' ? 0 : 30 }]}
    >
      <GlobalContext.Provider value={{ favData, setFavData }}>
        <NavigationContainer>
          <MyStack />
        </NavigationContainer>
      </GlobalContext.Provider>
    </SafeAreaView>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#101010',
    // marginHorizontal: 20,
  },
});
