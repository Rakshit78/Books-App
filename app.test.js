import React from 'react';
import { render, screen, fireEvent } from '@testing-library/react-native';

import App from './App';

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = render(<App />).toJSON();
    expect(tree).toMatchSnapshot();
    expect(1).toBe(1);
  });
});
