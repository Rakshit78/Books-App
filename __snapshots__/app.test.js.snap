// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> has 1 child 1`] = `
<RCTSafeAreaView
  style={
    [
      {
        "backgroundColor": "#101010",
        "flex": 1,
      },
      {
        "marginTop": 0,
      },
    ]
  }
/>
`;
